README.md

```diff
@@ -4,11 +4,11 @@ A collection of Go based sample code.
 
 ## Overview
 
-This repository host minimal assets develop for the Golang runtime. These samples are used in [Shipwrigh/Build](https://github.com/shipwright-io/build) for testing purposes.
+This repository host minimal assets develop for the Golang runtime. These samples are used in [Shipwright/build](https://github.com/shipwright-io/build) for testing purposes.
```

Comment:
can u keep the consistency of upper case on "build" , as in the later parts of this README, we write it on that way.

Reply:
or viceversa, if you want "Build" to be "build", not strong opinions, it should just be consistent across the README.

Reply:
Ha! Introducing a TYPO while trying to solve a TYPO. Classic IT.